lib/ruby/lib/alces/packager/config.rb

```diff
@@ -54,6 +54,10 @@ def dbroot(depot)
           File.expand_path(File.join(depot_path(depot),'etc'))
         end
 
+        def last_update_file
+          File.new(method_missing(:last_update_file))
```

Comment:
I think this might be better as `config[:last_update_file]` rather than going via `method_missing`.